thunder/benchmarks/targets.py

```diff
@@ -937,3 +938,42 @@ def test_interpreter_nanogpt_gpt2_fwd(benchmark, executor: Callable):
     fn = wrap_for_benchmark(fn)
 
     benchmark.pedantic(fn, setup=setup, rounds=5, warmup_rounds=1)
+
+
+#
+# vision benchmarks
+#
+
+
+@pytest.mark.parametrize(
+    "executor,",
+    fwd_executors,
+    ids=fwd_executor_ids,
+)
+def test_resnet50_fwd(benchmark, executor: Callable):
+    b = ResNet50Benchmark((64, 3, 224, 224), device="cuda:0", dtype=torch.bfloat16, requires_grad=False)
+
+    setup = make_setup(b)
+    fn = executor(b)
+    fn = wrap_for_benchmark(fn)
+
+    benchmark.pedantic(fn, setup=setup, rounds=5, warmup_rounds=1)
+
+
+@pytest.mark.parametrize(
+    "executor,",
+    (
+        torch_fwd_bwd,
+        torchcompile_fwd_bwd,
+        thunder_fwd_bwd,
+    ),
+    ids=fwd_executor_ids,
+)
+def test_resnet50_grad(benchmark, executor: Callable):
```

Comment:
It's more informative to split the "grad" benchmarks into forward and backward to be benchmarked separately. Here's the example of how this can be done: https://github.com/Lightning-AI/lightning-thunder/pull/148/files#diff-d84676ce7e46e2c100961abdf538428eff8cfefc16858b4caef8a1206d681cebR260